aesthetic.css

```diff
@@ -104,6 +106,7 @@ h2,
 h3 {
   margin-bottom: 0.6rem;
   letter-spacing: -0.06rem;
+  line-height: 2.6rem;
```

Comment:
Spacing between lines for headings was really poor on mobile, this increases the space when headings drop to multiple lines for nicer readability.